stdlib/builtins.pyi

```diff
@@ -1651,3 +1651,19 @@ class ResourceWarning(Warning): ...
 
 if sys.version_info >= (3, 10):
     class EncodingWarning(Warning): ...
+
+if sys.version_info >= (3, 11):
+    _SplitCondition = type[BaseException] | tuple[type[BaseException, ...]] | Callable[[BaseException], bool]
+    class BaseExceptionGroup(BaseException):
+        def __new__(cls, __message: str, exceptions: Sequence[BaseException]) -> BaseExceptionGroup | ExceptionGroup: ...
```

Comment:
According to the PEP, `exceptions` is also positional-only. Is the implementation different?

Reply:
Oops, my mistake.
```suggestion
        def __new__(cls, __message: str, __exceptions: Sequence[BaseException]) -> BaseExceptionGroup | ExceptionGroup: ...
```